src/runtime/virtcontainers/virtiofsd.go

```diff
@@ -34,13 +34,25 @@ var (
 	errVirtiofsdSocketPathEmpty    = errors.New("virtiofsd socket path is empty")
 	errVirtiofsdSourcePathEmpty    = errors.New("virtiofsd source path is empty")
 	errVirtiofsdSourceNotAvailable = errors.New("virtiofsd source path not available")
+
+	errUnimplemented = errors.New("unimplemented")
 )
 
-type Virtiofsd interface {
+type VirtiofsDaemon interface {
 	// Start virtiofsd, return pid of virtiofsd process
 	Start(context.Context, onQuitFunc) (pid int, err error)
 	// Stop virtiofsd process
 	Stop(context.Context) error
+	// Add a submount to the virtiofs mountpoint
```

Comment:
Should these `virtiofs` change to `nydus` or `rafs` will be more suitable?

Reply:
Maybe not, as `rafs` is submount of `virtiofs` mountpoint,  I think "Add a submount rafs to the virtiofs mountpoint" would be better?